deploy-agent/deployd/common/utils.py

```diff
@@ -219,18 +219,34 @@ def get_info_from_facter(keys):
         log.error("Failed to get info from facter by keys {}".format(keys))
         return None
 
-
-def get_container_health_info(key):
+    
+def get_container_health_info(commit):
     try:
-        log.info(f"Get health info for container {key}")
-        cmd = ['docker', 'inspect', '-f', '{{.State.Health.Status}}', key]
+        log.info(f"Get health info for container with commit {commit}")
+        result = ""
+        cmd = ['docker', 'ps', '--format', '{{.Image}};{{.Names}}']
```

Comment:
can you add UT for this method?

Reply:
it is not easy to add UT for this method.
I can show you the database data if that is ok for you ^^

```
mysql> select * from agents where host_name="dev-yaqinli" and container_health_status!="";
+-------------+------------------------+------------------------+---------------+--------+-----------+---------------+---------------+---------------+-------------+-------------+------------+--------------+---------------------+-------------------+------------------+--------------------------------------------------+
| host_name   | env_id                 | deploy_id              | deploy_stage  | state  | status    | start_date    | last_update   | last_operator | last_err_no | last_err_id | fail_count | first_deploy | host_id             | first_deploy_time | stage_start_date | container_health_status                          |
+-------------+------------------------+------------------------+---------------+--------+-----------+---------------+---------------+---------------+-------------+-------------+------------+--------------+---------------------+-------------------+------------------+--------------------------------------------------+
| dev-yaqinli | 2XwshUOHR36DGEAH4PzBwg | 3zZuGjXbReqgsgwkAFcQPQ | SERVING_BUILD | DELETE | SUCCEEDED | 1691714754494 | 1697676388311 | system        |           0 | NULL        |          0 |            0 | i-0dd4b6ce5062c36f0 |     1691714754494 |    1697676388311 | mcrouter:healthy;                                |
| dev-yaqinli | gu2DEmL4Tv2XOqDKFc7Axw | rXqFmaubTIGROjX0XT4VAg | SERVING_BUILD | DELETE | SUCCEEDED | 1691714754494 | 1697676388311 | system        |           0 | NULL        |          0 |            0 | i-0dd4b6ce5062c36f0 |     1691714754494 |    1697676388311 | ratelimit-A:healthy;                             |
| dev-yaqinli | inbDfs6KQICpRRfuKQw_bQ | IS_sXYi4RyGfHAojyXllcw | SERVING_BUILD | DELETE | SUCCEEDED | 1691714754494 | 1697676388311 | system        |           0 | NULL        |          0 |            0 | i-0dd4b6ce5062c36f0 |     1691714754494 |    1697676388311 | envoy-A:healthy;envoy-endpoint-register:healthy; |
| dev-yaqinli | pZwvW9faQTW7vD19ER72Gg | E8U5leLLSmmmAtkiqqy-kQ | SERVING_BUILD | DELETE | SUCCEEDED | 1691714754494 | 1697676388311 | system        |           0 | NULL        |          0 |            0 | i-0dd4b6ce5062c36f0 |     1691714754494 |    1697676388311 | tcollector:healthy;                              |
+-------------+------------------------+------------------------+---------------+--------+-----------+---------------+---------------+---------------+-------------+-------------+------------+--------------+---------------------+-------------------+------------------+--------------------------------------------------+
```